<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Doctor</class>
 <widget class="QWidget" name="Doctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>771</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#1A1627;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Image">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>141</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid;
border-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_name">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>Ad</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ID">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>511</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="title">
    <string>Hastalar</string>
   </property>
   <widget class="QTextBrowser" name="txt_allAppointment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_mid_a">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_mid_b">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_patientID">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Success">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_recete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_txt_recete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>Re√ßete</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_output">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_txt_output">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
